src/util/any.hpp

```diff
@@ -16,6 +16,9 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+#ifndef REALM_ANY_HPP
```

Comment:
We probably want to add tests for `Any` now that we're relying on its behavior in production code, rather than just tests.